opennms-config/src/main/java/org/opennms/netmgt/filter/JdbcFilterDao.java

```diff
@@ -458,6 +458,11 @@ public String getNodeMappingStatement(final String rule) throws FilterParseExcep
         columns.append(m_databaseSchemaConfigFactory.addColumn(tables, "nodeID"));
         columns.append(", " + m_databaseSchemaConfigFactory.addColumn(tables, "nodeLabel"));
 
+        //added table assets column
+        columns.append(", " + m_databaseSchemaConfigFactory.addColumn(tables, "city"));
+        columns.append(", " + m_databaseSchemaConfigFactory.addColumn(tables, "department"));
+        columns.append(", " + m_databaseSchemaConfigFactory.addColumn(tables, "address1"));
+
```

Comment:
we can't keep on adding these columns as we desire. Also the search term was used in asset description.